ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -53,6 +52,33 @@ module Json = struct
   let merge_maps m maps =
     List.fold_left (fun acc map -> StringMap.union choose_enum acc map) m maps
 
+  let rec get_fp_type ty =
```

Comment:
I understand the `fp` is for `function_partial_type`.
But after looking at the usage, it looks like `type in function name as a suffix`. And it is used only as a suffix of function name, how about `func_name_suffix`.